### ISSUE TITLE
upgrade to use node 14, replace deprecated new Buffer() by Buffer.from()

### ISSUE THREAD

**Issue by kevinhuang12345678** [msiebuhr/node-statsd-client#90]
workflow-regression_1  |      TypeError: Cannot convert a Symbol value to a string
workflow-regression_1  |       at Socket.<anonymous> (node_modules/zone.js/dist/zone-node.js:1665:52)
workflow-regression_1  |       at enqueue (dgram.js:498:10)
workflow-regression_1  |       at Socket.send (dgram.js:633:5)
workflow-regression_1  |       at /opt/haven/app/node_modules/statsd-client/lib/EphemeralSocket.js:182:22
workflow-regression_1  |       at EphemeralSocket._createSocket (node_modules/statsd-client/lib/EphemeralSocket.js:106:16)
workflow-regression_1  |       at EphemeralSocket._send (node_modules/statsd-client/lib/EphemeralSocket.js:174:10)
workflow-regression_1  |       at EphemeralSocket._flushBuffer (node_modules/statsd-client/lib/EphemeralSocket.js:150:10)
workflow-regression_1  |       at EphemeralSocket._enqueue (node_modules/statsd-client/lib/EphemeralSocket.js:138:14)
workflow-regression_1  |       at EphemeralSocket.send (node_modules/statsd-client/lib/EphemeralSocket.js:161:14)
workflow-regression_1  |       at StatsDClient.counter (node_modules/statsd-client/lib/statsd-client.js:82:18)
workflow-regression_1  |       at StatsDClient.increment (node_modules/statsd-client/lib/statsd-client.js:91:10)

suggest code change, from:
EphemeralSocket.prototype._send = function _send(data) {
    debug("_send(", data, ")");
    // If we don't have a socket, or we have created one but it isn't
    // ready yet, we need to enqueue data to send once the socket is ready.
    var that = this;

    this._createSocket(function () {
        that._socketUsed = true;

        // Create message
        var message = new Buffer(data);

        debug(message.toString());

        that._socket.send(message, 0, message.length, that._port, that._hostname);
    });
};

to:

EphemeralSocket.prototype._send = function _send(data) {
    debug("_send(", data, ")");
    // If we don't have a socket, or we have created one but it isn't
    // ready yet, we need to enqueue data to send once the socket is ready.
    var that = this;

    this._createSocket(function () {
        that._socketUsed = true;

        // Create message
        var message = Buffer.from(data);

        debug(message.toString());

        that._socket.send(message.toString(), 0, message.length, that._port, that._hostname);
    });
};

**Comment by papandreou** [comment]
@kevinhuang12345678, how about creating a pull request? It's much easier to understand and apply the change you're suggesting that way 🤗

**Comment by kevinhuang12345678** [comment]
@papandreou i do not have access to push my branch to create a pull request. :(

**Comment by papandreou** [comment]
You can fork the repository to your own account using this button:

<img width="959" alt="Screenshot 2021-04-20 at 00 40 40" src="https://user-images.githubusercontent.com/373545/115312199-32c3c800-a171-11eb-97b6-8f1315b3c478.png">

Then you can add the forked repository as a remote:

```sh
git remote add kevinhuang12345678 git@github.com:kevinhuang12345678/node-statsd-client
```

and then you can push your branch with the change to the fork (change `HEAD` to the branch name if it's not the current branch):

```sh
git push -u kevinhuang12345678 HEAD
```

Once that's done, you can create the pull request from your fork.

**Comment by msiebuhr** [comment]
Fixed per #91
